src/backend/os.c

```diff
@@ -710,7 +710,7 @@ char *file_8dot3name(const char *filename)
     char *buf;
     int i;
 
-    h = FindFirstFile(filename,&fileinfo);
+    h = FindFirstFileA(filename,&fileinfo);
```

Comment:
*A versions are basically deprecated junk on Windows. Do not use unless absolutely needed.

Reply:
P.S.: NT kernel is UTF-16 throughout.